data/Publications.json

```diff
@@ -6852,13 +6852,13 @@
         "description": "ISOCAM observations of the Chamaeleon I dark cloud"
     },
     {
-        "publication": "Lópe04",
+        "publication": "L\u00f3pe04",
```

Comment:
We've got some unicode in here and I'm not sure what to do. @dr-rodriguez , thoughts? I feel like we've dealt with this before but I don't remember the discussion.

Reply:
This should be fixed now, but we will need to merge the latest changes from main into this branch and re-ingest the RVs